src/panels/config/logs/error-log-card.ts

```diff
@@ -232,27 +240,40 @@ class ErrorLogCard extends LitElement {
                   `ui.panel.config.logs.${this._wrapLines ? "full_width" : "wrap_lines"}`
                 )}
               ></ha-icon-button>
-              ${!this._streamSupported || this._error
+              ${!streaming
```

Comment:
```suggestion
              ${!streaming  || this._error
```

Refresh should also be shown on error